docs/Jenkins & Docker.txt

```diff
@@ -0,0 +1,24 @@
+Jenkins and Docker are installed on the remote server so there is no need to have them on your local machine.
+To be able to operate them you have to receive your accounts. Once they are created for you ->
+Login to -web UI     https://bonnee.eu/ci/ 
+         -CLI        ssh accountName@bonnee.eu   
+         
+--Jenkins--
+The current Bonnie-Jenkins build is using Jenkinsfile located in the parent folder (Hexagonal).
+The branch source is: "dockerize-components" from "https://github.com/tamas-csanyi-cgi/bonnie" repository.
```

Comment:
We can delete this line, after https://github.com/tamas-csanyi-cgi/bonnie/pull/52 is merged then we'll have Jenkinsfile in master too, to furher branches will inherit this.